crates/bench/benches/subscription.rs

```diff
@@ -148,6 +151,57 @@ fn eval(c: &mut Criterion) {
         })
     });
 
+    //To profile this benchmark for 30s
+    //samply record -r 10000000 cargo bench --bench=subscription --profile=profiling -- select-protobuf --exact --profile-time=30
+    c.bench_function("select-protobuf", |b| {
+        // A passthru executed independently of the database.
+        let select_lhs = "select * from footprint";
+        let auth = AuthCtx::for_testing();
+        let tx = db.begin_tx();
+        let query_lhs = compile_read_only_query(&db, &tx, &auth, select_lhs).unwrap();
+        let query = ExecutionSet::from(query_lhs);
+        let out = query.eval_mem_table(&db, &tx).unwrap();
+
+        let out = DatabaseTableUpdate {
+            table_id: 0.into(),
+            table_name: "test".into(),
+            ops: out
+                .into_iter()
+                .flat_map(|x| x.data.into_iter().map(TableOp::insert))
+                .collect(),
+        };
+
+        b.iter_batched(
+            || out.clone(),
+            |out| {
+                let _ = TableUpdate::from(out);
+            },
+            criterion::BatchSize::PerIteration,
+        );
+    });
+
+    //To profile this benchmark for 30s
+    //samply record -r 10000000 cargo bench --bench=subscription --profile=profiling -- select-bsatn --exact --profile-time=30
+    c.bench_function("select-bsatn", |b| {
```

Comment:
I'd rather the benchmarks cover both bsatn and protobuf instead of individual benchmarks for each.

Reply:
Ok, I reworked the benches adding the `send_message` that executes the full pipeline and removing the different